docker-compose.yaml

```diff
@@ -1,28 +1,67 @@
 version: '3'
 
+# Generate self-signed certs with:
+# openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes
+# openssl req -new -key key.pem -x509 -days 1000 -out CA-cert.pem
```

Comment:
What is the different between these 2 certificates?

Reply:
A CA certificate is a certificate from a "Certificate Authority". It can be used to generate other certificates. Of course, since this is self-signed and generated, most web browsers would throw up a lot of red flags. If the services using these certificates were to be opened up the rest of the world, I'd probably be using certificates from Let's Encrypt, or something like that. But since this is all internal, self-signed certificates are OK. They can still be used to encrypt traffic, they're just not as secure-looking as certificates from an actual CA.